stan/math/prim/meta/is_vector.hpp

```diff
@@ -100,6 +100,11 @@ STAN_ADD_REQUIRE_CONTAINER(eigen_col_vector, is_eigen_col_vector,
 template <typename T>
 struct is_col_vector : internal::is_col_vector_impl<T> {};
 
+STAN_ADD_REQUIRE_UNARY(col_vector, is_col_vector,
```

Comment:
I needed a `col_vector` that wasn't necessarily an `eigen_col_vector` so I added these. Are there tests I should add for these?

Reply:
These are in #2106 now